src/QuantumOpticsBase.jl

```diff
@@ -28,6 +28,11 @@ export Basis, GenericBasis, CompositeBasis, basis,
         #operators_lazytensor
                 LazyTensor, lazytensor_use_cache, lazytensor_clear_cache,
                 lazytensor_cachesize, lazytensor_disable_cache, lazytensor_enable_cache,
+        #time_dependent_operators
+                AbstractTimeDependentOperator, TimeDependentSum, is_const, set_time!,
+                current_time, coefficients, suboperators, eval_coefficient,
```

Comment:
I am a bit on the fence about exporting `coefficients`, `suboperators`, and a lot other of these helpers. Could we consider keeping as many of them as possible as private, in case we want to modify something as we are evaluating SciMLOperators?